Xamarin.PropertyEditing/ViewModels/CreateBindingViewModel.cs

```diff
@@ -674,5 +685,18 @@ IReadOnlyList<object> IProvidePath.GetItemParents (object item)
 				throw new ArgumentException();
 			}
 		}
+
+		private readonly RelayCommand requestAddValueConverterCommand;
+		public ICommand RequestAddValueConverter => this.requestAddValueConverterCommand;
+
+		private bool CanRequestAddValueConverter ()
+		{
+			return TargetPlatform.BindingProvider != null;
```

Comment:
@ermau Any other checks required to validate if the `AddValueConverter` button should be enabled when the BindingSource changes?